### DISCUSSION TITLE
Mpv Android developer support

### DISCUSSION THREAD

**Discussion by Sensex00** [mpv-player/mpv#14257] [Q&A]
Mpv Android dire need of developers.
launch in 2016 but it still misses some of the basic features like volume boost up to 200% and external font for subtitles support till now and lack ui because only one person is maintaing the repository

My ask is to please contribute to the MPV Android if you can

**Comment by llyyr** [comment]
> My ask is to please contribute to the MPV Android if you can

How much are you paying?

**Reply by Sensex00** [reply]
> > My ask is to please contribute to the MPV Android if you can
> 
> How much are you paying?

Hey i am not forcing anybody I am just encouraging to contribute on free will that all

**Comment by hooke007** [comment]
Most libmpv frontend is maintained by only one person. So do most of repos in github.
Mpv's discussion shouldn't be the place for advertising for other repos.

**Reply by Sensex00** [reply]
You are saying right but on android only one mpv frontend MPV Android is available my intention was good and clear but I guess I should close this discussion so I will soon
